src/main/java/org/myrobotlab/service/UltrasonicSensor.java

```diff
@@ -74,7 +75,8 @@ public UltrasonicSensor(String n, String id) {
   // change units, sample rate, etc
   // FIXME - NOT SERVICE .. possibly name or interface but not service
   public void addRangeListener(Service service) {
-    addListener("publishRange", service.getName(), "onRange");
+    addListener(""

```

Comment:
What is "ge"  ?

Reply:
what the hell ? ... pre-coffee mess ... thanks for the catch